conf/otherassetsdisposal.routes

```diff
@@ -34,4 +34,14 @@ POST       /:srn/change-total-consideration-sale-asset/:assetIndex/:disposalInde
 GET         /:srn/type-of-asset-buyer/:assetIndex/:disposalIndex                                   controllers.nonsipp.otherassetsdisposal.TypeOfAssetBuyerController.onPageLoad(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = NormalMode)
 POST        /:srn/type-of-asset-buyer/:assetIndex/:disposalIndex                                   controllers.nonsipp.otherassetsdisposal.TypeOfAssetBuyerController.onSubmit(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = NormalMode)
 GET         /:srn/change-type-of-asset-buyer/:assetIndex/:disposalIndex                            controllers.nonsipp.otherassetsdisposal.TypeOfAssetBuyerController.onPageLoad(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = CheckMode)
-POST        /:srn/change-type-of-asset-buyer/:assetIndex/:disposalIndex                            controllers.nonsipp.otherassetsdisposal.TypeOfAssetBuyerController.onSubmit(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = CheckMode)
\ No newline at end of file
+POST        /:srn/change-type-of-asset-buyer/:assetIndex/:disposalIndex                            controllers.nonsipp.otherassetsdisposal.TypeOfAssetBuyerController.onSubmit(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = CheckMode)
+
+GET         /:srn/individual-name-of-asset-buyer/:assetIndex/:disposalIndex                             controllers.nonsipp.otherassetsdisposal.IndividualNameOfAssetBuyerController.onPageLoad(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = NormalMode)
+POST        /:srn/individual-name-of-asset-buyer/:assetIndex/:disposalIndex                             controllers.nonsipp.otherassetsdisposal.IndividualNameOfAssetBuyerController.onSubmit(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = NormalMode)
+GET         /:srn/change-individual-name-of-asset-buyer/:assetIndex/:disposalIndex                      controllers.nonsipp.otherassetsdisposal.IndividualNameOfAssetBuyerController.onPageLoad(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = CheckMode)
+POST        /:srn/change-individual-name-of-shares-buyer/:assetIndex/:disposalIndex                      controllers.nonsipp.otherassetsdisposal.IndividualNameOfAssetBuyerController.onSubmit(srn: Srn, assetIndex: Max5000, disposalIndex: Max50, mode: Mode = CheckMode)
```

Comment:
Should be `individual-name-of-asset-buyer` instead of `individual-name-of-shares-buyer`